3p/messaging.js

```diff
@@ -14,6 +14,52 @@
  * limitations under the License.
  */
 
+import './polyfills';
+import {dev} from '../src/log';
+
+/** @const */
+const AMP_MESSAGE_PREFIX = 'amp-';
+
+/** @enum {string} */
+export const MessageType = {
```

Comment:
I thought about this yesterday. But we don't need the `MessageType` object in ampcontext.js for inabox-host code. Should we really combine them or just move `MessageType` in ampcontext.js to messaging.js, while keep them in different object?